tracer/test/Datadog.Trace.Tests/Agent/StatsAggregatorTests.cs

```diff
@@ -456,7 +456,8 @@ public void SubscribeToChanges(Action<AgentConfiguration> callback)
                              telemetryProxyEndpoint: "telemetryProxyEndpoint",
                              tracerFlareEndpoint: "tracerFlareEndpoint",
                              clientDropP0: true,
-                             spanMetaStructs: true));
+                             spanMetaStructs: true,
+                             spanEvents: false));
```

Comment:
I suggest turning these `spanEvents: true` everywhere to make sure that we test the new code path